### PR TITLE
Feat/profile

### PR DESCRIPTION
**Pull request by Dikus21**
- Remove aktivitas keuangan chart and download section
- Add dropdown card change selection
- Add current card and user info section
- Modified scorecard component to accept no rekening and click to copy
 no rekening
- Synchronize card change on profile with homepage
- Add environtment variables for api base url

### CODE REVIEW

File: src/services/authService.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const loginUser = async (username: string, password: string): Promise<str
         if (localStorage.getItem("activeSession") === "true") {
             throw new Error("Ada sesi aktif di tab lain. Silakan tutup tab tersebut terlebih dahulu.");
         }
-        const response = await fetch("https://cautious-noelyn-ridho-71c54445.koyeb.app/api/v1/auth/login", {
+        const response = await fetch(import.meta.env.VITE_API_BASE_URL + "api/v1/auth/login", {
```
Comment (Jeremiapane) [inline src/services/authService.ts:6]: ini conflict ya mas, bisa di solve aja pake yang .env .

**Review by Jeremiapane — CHANGES_REQUESTED** [review]
yang conflict di solve dulu ya mas abis itu di merge, thank you

**Comment by Dikus21** [comment]
Oke mas udah saya resolve, bisa di cek lagi

File: src/services/authService.ts (hunk 1 above)
Comment (Dikus21) [reply src/services/authService.ts:6]: Oke mas udah saya resolve, bisa di cek lagi @Jeremiapane

**Review by Jeremiapane — APPROVED** [review]
nice work mas 🔥

**Pull request merged**